ffs-shared/env/src/commonMain/kotlin/env/EnvConst.kt

```diff
@@ -8,3 +8,4 @@ const val ENV_USER_EMAIL = "user.email"
 const val ENV_DEVICE_NAME = "device.name"
 const val ENV_DEVICE_OS = "device.os"
 const val ENV_DEVICE_LOCALE = "device.locale"
+const val ENV_DEVICE_IP_ADDRESS = "device.ip_address"
```

Comment:
```suggestion
const val ENV_DEVICE_IP_ADDRESS = "device.ip-address"
```

`internal.rollout-id` is the only other multi-word key, and it uses dashes.